client/src/components/Navbar/Navbar.jsx

```diff
@@ -1,17 +1,21 @@
 import { useState } from "react";
 import Search from "../Search/Search";
+import Burgermenu from "./Burgermenu";
 import "./Navbar.scss";
 
-
 function Navbar() {
   const [isSearchVisible, setSearchVisible] = useState(false);
+  const [hamburgerOpen, setHamburgerOpen] = useState(false);
```

Comment:
rename it burgerMenu no ?

Reply:
Je trouve ça plus clair avec hamburger open, je ne peux pas laisser ça plutôt ?